### DISCUSSION TITLE
How to disable clickable rows on table builder using conditional

### DISCUSSION THREAD

**Discussion by andre-arc** [filamentphp/filament#7033] [Help]
i want to know how to disable clickable rows on table builder using conditional operator, so there is row on table that disable

**Comment by ArtMin96** [comment]
Docs: [Disabling clickable rows](https://filamentphp.com/docs/2.x/tables/getting-started#disabling-clickable-rows)
```php
protected function getTableRecordActionUsing(): ?Closure
{
    return null;
}
```
